src/x-scale-axis.ts

```diff
@@ -47,19 +47,29 @@ export class AxisTicksSetter {
    * Handles setting the tickFormatter function
    */
   handleAxisTicks() {
-    const { scaleType } = this.cle;
-    if (scaleType !== "continuous" && scaleType !== "categorical") {
+    // NOTE: used to customize the tick values for certain legend types
+    if (
+      this.cle.scaleType !== "continuous" &&
+      this.cle.scaleType !== "categorical" &&
```

Comment:
could also change these to
```suggestion
      (this.cle.scaleType === "discrete || this.cle.scaleType === "threshold") &&
```